### PR TITLE
Add function to count words in file

### LINKED ISSUE #4: [Student B] Count words
Check instructions from the doc

### PR DESCRIPTION
**Pull request by 1929940**
Add word counter
resolve #4 

- By Maciej Karczewski

### CODE REVIEW

File: AppMenu.cs [hunk 1]
```diff
@@ -64,6 +65,7 @@ public static void MainMenu()
                         {
                             try
                             {
+                                lettersInFile = CountWords();
                             }
                             catch (Exception)
```
Comment (Ostrokrzew) [inline AppMenu.cs]: Please, change exception type.

File: AppMenu.cs [hunk 2]
```diff
@@ -64,6 +65,7 @@ public static void MainMenu()
                         {
                             try
                             {
+                                lettersInFile = CountWords();
```
Comment (Ostrokrzew) [inline AppMenu.cs]: I think this var should be wordsInFile.

**Review by Ostrokrzew — CHANGES_REQUESTED** [review]

File: AppMenu.cs (hunk 2 above)
Comment (1929940) [reply AppMenu.cs]: done

File: AppMenu.cs (hunk 1 above)
Comment (1929940) [reply AppMenu.cs]: done

**Review by Ostrokrzew — APPROVED** [review]

**Review by Ostrokrzew — APPROVED** [review]

**Pull request merged**